api/api/settings.py

```diff
@@ -36,8 +36,8 @@
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
-    'django.contrib.staticfiles',
```

Comment:
Rut roh! What happened to `staticfiles`? I mean, if we don't need it then it's cool

Reply:
Looks like we need `staticfiles`:

![Screen Shot 2019-12-17 at 3 12 36 PM](https://user-images.githubusercontent.com/4991553/71030563-b08c4500-20df-11ea-9240-261fa62ff257.png)